apollo-backend/src/main/resources/db/migration/V20200309182111__Add_slave.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE `slave` (
```

Comment:
table name should be plural: slaves

Reply:
all other tables such as environment, service, etc. are in singular

Reply:
Singular: blocker_definition, deployable_version, deployment, environment, notification, service, stack
Plural: deployment_permissions, deployment_role_permissions, deployment_roles, groups, user_deployment_roles, users
Special: environments_in_stacks, services_in_stacks, users_blockers_override

Unfortunately there's no standard in this project, at Logz.io we use plural  and I think we should do that here too.